### DISCUSSION TITLE
Ubuntu 22.04 Requirement

### DISCUSSION THREAD

**Discussion by gatorback** [curzon01/mqtt2mysql#30] [General]
Before execution of python3 mysql module.   2 Lines for 22.04  libmysql requirement:

     sudo apt-get install python3-dev default-libmysqlclient-dev build-essential

     sudo apt install pkg-config

see: https://stackoverflow.com/a/76588599/4953146

**Comment by curzon01** [comment]
Good specific hint for Ubuntu 22.04
